guides/source/caching_with_rails.md

```diff
@@ -31,6 +31,8 @@ the relevant `config/environments/*.rb` file:
 config.action_controller.perform_caching = true
 ```
 
+NOTE: Changing the value of `config.action_controller.perform_caching` will only have an effect on the caching provided by the Action Controller component. For instance, it will not impact low-level caching, that we address [below](#low-level-caching).
```

Comment:
Didn't see that. Since many other places in the same doc are not wrapped (see the info) I didn't see the need for it at first. It's done now.